packages/mask/src/plugins/dHEDGE/base.tsx

```diff
@@ -5,7 +5,7 @@ export const base: Plugin.Shared.Definition = {
     ID: DHEDGE_PLUGIN_ID,
     name: { fallback: 'dHEDGE' },
     description: { fallback: 'Decentralized hedge funds on Ethereum.' },
-    publisher: { name: { fallback: 'iRhonin' }, link: 'https://github.com/iRhonin' },
+    publisher: { name: { fallback: 'Mask Network' }, link: 'https://mask.io/' },
```

Comment:
?

Reply:
![image](https://user-images.githubusercontent.com/72531217/168764206-3f6ba3ce-29d7-4d51-9c85-8e9975282254.png)

Reply:
You don't need to follow the Figma about the publisher. It's customizable.